content/en/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -78,6 +74,14 @@ Pipelines don't need any specific changes to authenticate to Google Cloud. With
 However, existing pipelines that use [use_gcp_secret kfp sdk operator](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.extensions.html#kfp.gcp.use_gcp_secret) need to remove the `use_gcp_secret` usage to use the bound GSA.
 You can also continue to use `use_gcp_secret` in a cluster with Workload Identity enabled and `use_gcp_secret` will take precedence for those workloads.
 
+#### Cluster setup to use Workload Identity for Full Kubeflow
+
+Starting from Kubeflow 1.1, Kubeflow Pipelines [supports multi-user isolation](/docs/pipelines/multi-user/). Therefore, pipeline runs are executed in user namespaces using the `default-editor` KSA. The `default-editor` KSA is auto-bound to the GSA specified in the user profile, which defaults to a shared GSA `${KFNAME}-user@${PROJECT}.iam.gserviceaccount.com`.
+
+If you want to bind `default-editor` KSA with a chosen GSA for a specific namespace, refer to [In-cluster authentication to Google Cloud](/docs/gke/authentication/#in-cluster-authentication).
+
+Additionally, the Kubeflow Pipelines UI / Visualization / TensorBoard servers also deploy instances in your user namespace using the `default-editor` KSA. Therefore, to [visualize results in the Pipelines UI](/docs/pipelines/sdk/output-viewer/), they can fetch artifacts in Google Cloud Storage using permissions of the same GSA you configured for this namespace.
```

Comment:
Are artifacts always on GCS, or can they be in the cluster's storage?

Reply:
It's optional, they can be on GCS or inline or S3, but they cannot be on cluster's storage currently